src/main/java/com/blamejared/contenttweaker/api/fluids/IIsCotFluid.java

```diff
@@ -23,4 +23,14 @@
      * Sets block of the fluid, shouldn't be overridden if the fluid is flowing type
      */
     default void setFluidBlock(FlowingFluidBlock flowingFluidBlock) {}
+
+    /**
+     * @return if the fluid is molten
+     */
+    boolean isMolten();
+
+    /**
+     * @return if the fluid should be tagged as water or lava.
+     */
+    boolean isTagged();
```

Comment:
Adding things that are not equivalent to water or lava to the water and lava tags are extremely incorrect things to be doing.... Though I do think it would make some sense to automatically add fluids (both still and flowing) to a tag of the same fluid name as the fluid itself is so that say if a mod adds honey and CoT is adding honey they are equivalent via `forge:honey` (though at that point not sure why someone would want to add it with CoT if their pack already has it)

Reply:
You are right. But I need water/lava tag behavior. For example, pushing (water tag) or burning (lava tag) entities.

Reply:
Which is not at all the correct way of doing things and is a massive hack (that imo has no place in CoT) and is why @Lanse505 and a few others are working on creating some proper hooks in forge so that behaviors of custom fluids can be defined properly.

Reply:
Fair enough. I'm going to remove tag stuff, waiting for nice forge hooks.